app/config/FrontendAppConfig.scala

```diff
@@ -110,6 +110,10 @@ class FrontendAppConfig @Inject()(val config: ServicesConfig,
     appendLanguage(portalHost + config.getString(s"urls.external.portal.$key")).format(args: _*)
   }
 
+  def getVrsUrl(key: String)(implicit request: Request[_]): String = {
+    appendLanguage(config.getString(s"urls.vat-registration-service.$key"))
+  }
+
```

Comment:
This already exists on lines 175 to 176. The append language isn't necessary, VRS uses the language cookie so won't use the query string